Integrations/Slack/Slack_test.py

```diff
@@ -2079,54 +2149,239 @@ def test_send_topic_no_args_no_investigation(mocker):
 
     # Arrange
     with pytest.raises(InterruptedError):
-        Slack.slack_set_channel_topic()
+        Slack.set_channel_topic()
 
     err_msg = return_error_mock.call_args[0][0]
 
     # Assert
     assert Slack.get_conversation_by_name.call_count == 0
-    assert err_msg == 'No channel was provided.'
+    assert err_msg == 'Channel not found - the Demisto app needs to be a member of the channel in order to look it up.'
 
 
-def test_send_to_user_lowercase(mocker):
+def test_invite_users(mocker):
     import Slack
 
     # Set
 
-    def users_list(**kwargs):
-        return {'members': json.loads(USERS)}
+    mocker.patch.object(demisto, 'args', return_value={'channel': 'general', 'users': 'spengler, glinda'})
+    mocker.patch.object(demisto, 'investigation', return_value={'id': '681'})
+    mocker.patch.object(demisto, 'getIntegrationContext', side_effect=get_integration_context)
+    mocker.patch.object(demisto, 'setIntegrationContext', side_effect=set_integration_context)
+    mocker.patch.object(Slack, 'get_conversation_by_name', return_value={'id': 'C012AB3CD'})
+    mocker.patch.object(Slack, 'invite_users_to_conversation')
+    mocker.patch.object(demisto, 'results')
 
-    def conversations_list(**kwargs):
-        return {'channels': json.loads(CONVERSATIONS)}
+    # Arrange
+    Slack.invite_to_channel()
+
+    send_args = Slack.invite_users_to_conversation.call_args[0]
+    success_results = demisto.results.call_args[0]
+
+    # Assert
+    assert Slack.get_conversation_by_name.call_count == 1
+    assert Slack.invite_users_to_conversation.call_count == 1
+    assert success_results[0] == 'Successfully invited users to the channel.'
+    assert send_args[0] == 'C012AB3CD'
+    assert send_args[1] == ['U012A3CDE', 'U07QCRPA4']
+
+
+def test_invite_users_no_channel(mocker):
```

Comment:
lets add test to invite users, with no channel given and no mirror channel is found, and assert the error